libcontainer/container_linux.go

```diff
@@ -1060,17 +1060,12 @@ func (c *linuxContainer) currentState() (*State, error) {
 		ExternalDescriptors: externalDescriptors,
 	}
 	if pid > 0 {
-		for _, ns := range c.config.Namespaces {
-			state.NamespacePaths[ns.Type] = ns.GetPath(pid)
-		}
```

Comment:
I believe if you delete this code and the `Path` was previously set in one of the `c.config.Namespaces`.. we are no longer storing said path in the state.